### PR TITLE
318 update for tes compliance

### PR DESCRIPTION
**Pull request by vsmalladi**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/fetchngs/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/fetchngs _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [PR #303](https://github.com/nf-core/fetchngs/pull/303) - Update wget container for SRA_FASTQ_FTP from 1.20.1 to 1.21.4
 - [PR #305](https://github.com/nf-core/fetchngs/pull/305) - Update module sratools/prefetch for reliable download integrity check
 - [PR #316](https://github.com/nf-core/fetchngs/pull/316) - Use nf-core/setup-nf-test to install nf-test from cache during CI/CD
+- [PR #320](https://github.com/nf-core/fetchngs/pull/20) - Update module multiqc_mappings_config and sratools/prefetch for TES compliance
```
Comment (maxulysse) [inline CHANGELOG.md]:
```suggestion
- [PR #320](https://github.com/nf-core/fetchngs/pull/320) - Update module multiqc_mappings_config and sratools/prefetch for TES compliance
```

**Comment by maxulysse** [comment]
Linting is failing, but it's ok, I'll finish working up on TEMPLATE sync and then merge this one.
sorry for the delay @vsmalladi

**Comment by vsmalladi** [comment]
Great thanks @maxulysse

**Comment by maxulysse** [comment]
That looks good to me, but I'm asking an extra set of eyes for safety because of the updates in all the tests

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [PR #303](https://github.com/nf-core/fetchngs/pull/303) - Update wget container for SRA_FASTQ_FTP from 1.20.1 to 1.21.4
 - [PR #305](https://github.com/nf-core/fetchngs/pull/305) - Update module sratools/prefetch for reliable download integrity check
 - [PR #316](https://github.com/nf-core/fetchngs/pull/316) - Use nf-core/setup-nf-test to install nf-test from cache during CI/CD
+- [PR #320](https://github.com/nf-core/fetchngs/pull/320) - Update module multiqc_mappings_config and sratools/prefetch for TES compliance
```
Comment (maxulysse) [inline CHANGELOG.md:15]:
```suggestion
- [PR #320](https://github.com/nf-core/fetchngs/pull/320) - Update module multiqc_mappings_config and sratools/prefetch for TES compliance
- [PR #320](https://github.com/nf-core/fetchngs/pull/320) - Refactor workflow tests
```

File: modules/local/multiqc_mappings_config/main.nf [hunk 3]
```diff
@@ -10,7 +10,7 @@ process MULTIQC_MAPPINGS_CONFIG {
     path csv
 
     output:
-    path "*yml"        , emit: yml
+    path "*config.yml"        , emit: yml
     path "versions.yml", emit: versions
```
Comment (JoseEspinosa) [inline modules/local/multiqc_mappings_config/main.nf]:
```suggestion
    path "*config.yml" , emit: yml
    path "versions.yml", emit: versions
```

**Review by JoseEspinosa — APPROVED** [review]
LGTM! Just a small formatting suggestion

**Comment by vsmalladi** [comment]
Awesome. I think I can update the rest for tea compliance as well

**Pull request merged**